### ISSUE TITLE
Add another handle to the accessible toilet

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1151]
We had another wheelchair user in the space today, and they suggested the accessible toilet could be improved if it had a handle on the other side (right-hand side as you face it) of the toilet.  Looks like there should be room for one of the ones that's vertical and then lifts up and folds down to give people more flexibility over how they use it.

**Comment by johnmckerrell** [comment]
Few options on Screwfix:

https://www.screwfix.com/p/nymas-lift-lock-hinged-support-rail-white-800-x-200-x-35mm/2605x

https://www.screwfix.com/p/elderly-disabled-double-support-hinged-rail-w-leg-white-310-x-775-925-x/33062

https://www.screwfix.com/p/elderly-disabled-double-support-hinged-rail-steel-white-250-x-250-x/82759

Although the first of these is the cheapest it also claims the highest load so I'm not really sure why we should go with anything else. It doesn't include the fixtures though, not sure if it requires special screws or if standard long screws and rawl plugs will do.
